tiatoolbox/wsicore/wsireader.py

```diff
@@ -3007,6 +3011,13 @@ def __getitem__(self, index):
         while len(index) < len(self.axes):
             index = (*index, slice(None))
 
+        # toilet roll multiplex OME TIFF
```

Comment:
Maybe change to "separate planar configuration (AKA toilet roll)" to use the same terminology in spec documents.